pkg/audit/manager.go

```diff
@@ -403,13 +414,71 @@ func (am *Manager) auditResources(
 			}
 		}
 	}
+	// loop thru each subDir in output dir to get files
+	for i := 0; i < folderCount; i++ {
+		subDir := fmt.Sprintf("%d", i)
+		pDir := path.Join(*outputDir, subDir)
+		files, err := os.ReadDir(pDir)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+		for _, f := range files {
+			fileName := f.Name()
+			objNs := strings.Split(fileName, "_")[0]
```

Comment:
If this is why we are embedding the namespace in the filename, could we retrieve the namespace after we've called `am.readUnstructured()` and get it from the object itself?

Reply:
Per above comment, we are already making the api call to get namespace to test ns exclusion prior to saving the file. It makes more sense to me to save it as part of the file at that point instead.

Reply:
Except we are calling `nsCache.Get()` here anyway? Per above, we'd need to get the whole namespace object so we can evaluate the namespace selector.

Reply:
done